front/src/app/models/transaction.model.ts

```diff
@@ -1,3 +1,17 @@
+import {Address} from './address.model';
+import {Log} from 'web3-core';
+
+export class ProcessedLog {
+  index: number;
+  contract_address: string;
+  data: string;
+  removed: boolean;
+}
+
+export interface TxLog extends Log {
+  removed: boolean;
```

Comment:
Strange that this field is missing from the official library...

Reply:
yeah, no typo for that